airflow-postgres-app-services/azuredeploy.json

```diff
@@ -0,0 +1,227 @@
+            }
+        },
+        "databaseSkuName": {
+            "type": "string",
+            "defaultValue": "B_Gen5_1",
+            "allowedValues": [
+                "GP_Gen5_2",
+                "GP_Gen5_4",
+                "GP_Gen5_8",
+                "GP_Gen5_16",
+                "GP_Gen5_32",
+                "MO_Gen5_2",
+                "MO_Gen5_4",
+                "MO_Gen5_8",
+                "MO_Gen5_16",
+                "MO_Gen5_32",
+                "B_Gen5_1",
+                "B_Gen5_2"
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL sku name "
+            }
+        },
+        "databaseSkuSizeMB": {
+            "type": "int",
+            "allowedValues": [
+                102400,
+                51200,
+                10240,
+                5120
+            ],
+            "defaultValue": 5120,
+            "metadata": {
+                "description": "Azure database for PostgreSQL Sku Size "
+            }
+        },
+        "databaseSkuTier": {
+            "type": "string",
+            "defaultValue": "Basic",
+            "allowedValues": [
+                "GeneralPurpose",
+                "MemoryOptimized",
+                "Basic"
+            ],
+            "metadata": {
+                "description": "Azure database for PostgreSQL pricing tier"
+            }
+        },
+        "databaseskuFamily": {
+            "type": "string",
+            "defaultValue": "Gen5",
+            "metadata": {
+                "description": "Azure database for PostgreSQL sku family"
+            }
+        },
+        "postgresqlVersion": {
+            "type": "string",
+            "allowedValues": [
+                "9.5",
+                "9.6",
+                "10",
+                "11"
+            ],
+            "defaultValue": "11",
+            "metadata": {
+                "description": "PostgreSQL version"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        }
+    },
+    "variables": {
+        "databaseName": "[concat(parameters('airflowWebAppName'), 'database')]",
+        "serverName": "[concat(parameters('airflowWebAppName'), 'pgserver')]",
+        "hostingPlanName": "[concat(parameters('airflowWebAppName'), 'serviceplan')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2018-11-01",
+            "type": "Microsoft.Web/serverfarms",
+            "name": "[variables('hostingPlanName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "name": "[variables('hostingPlanName')]",
+                "workerSize": "1",
+                "reserved": true,
+                "numberOfWorkers": 1
+            },
+            "sku": {
```

Comment:
move this above properties

Reply:
Updated based on https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties